class UserAlreadyExistsError(Exception):
    """
    Raised when attempting to add a user with a username that already exists.
    """
    pass
